lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -59,57 +59,60 @@ class InspectorIssues extends FRGatherer {
    */
   async _getArtifact(networkRecords) {
     const artifact = {
-      /** @type {Array<LH.Crdp.Audits.MixedContentIssueDetails>} */
-      mixedContent: [],
       /** @type {Array<LH.Crdp.Audits.SameSiteCookieIssueDetails>} */
-      sameSiteCookies: [],
+      sameSiteCookieIssue: [],
+      /** @type {Array<LH.Crdp.Audits.MixedContentIssueDetails>} */
+      mixedContentIssue: [],
       /** @type {Array<LH.Crdp.Audits.BlockedByResponseIssueDetails>} */
-      blockedByResponse: [],
+      blockedByResponseIssue: [],
       /** @type {Array<LH.Crdp.Audits.HeavyAdIssueDetails>} */
-      heavyAds: [],
+      heavyAdIssue: [],
       /** @type {Array<LH.Crdp.Audits.ContentSecurityPolicyIssueDetails>} */
-      contentSecurityPolicy: [],
+      contentSecurityPolicyIssue: [],
+      /** @type {Array<LH.Crdp.Audits.SharedArrayBufferIssueDetails>} */
+      sharedArrayBufferIssue: [],
+      /** @type {Array<LH.Crdp.Audits.TrustedWebActivityIssueDetails>} */
+      twaQualityEnforcement: [],
+      /** @type {Array<LH.Crdp.Audits.LowTextContrastIssueDetails>} */
+      lowTextContrastIssue: [],
+      /** @type {Array<LH.Crdp.Audits.CorsIssueDetails>} */
+      corsIssue: [],
+      /** @type {Array<LH.Crdp.Audits.AttributionReportingIssueDetails>} */
+      attributionReportingIssue: [],
+      /** @type {Array<LH.Crdp.Audits.QuirksModeIssueDetails>} */
+      quirksModeIssue: [],
+      /** @type {Array<LH.Crdp.Audits.NavigatorUserAgentIssueDetails>} */
+      navigatorUserAgentIssue: [],
+      /** @type {Array<LH.Crdp.Audits.WasmCrossOriginModuleSharingIssueDetails>} */
+      wasmCrossOriginModuleSharingIssue: [],
+      /** @type {Array<LH.Crdp.Audits.GenericIssueDetails>} */
+      genericIssue: [],
       /** @type {Array<LH.Crdp.Audits.DeprecationIssueDetails>} */
-      deprecations: [],
+      deprecationIssue: [],
     };
 
-    for (const issue of this._issues) {
-      if (issue.details.mixedContentIssueDetails) {
-        const issueDetails = issue.details.mixedContentIssueDetails;
-        const issueReqId = issueDetails.request && issueDetails.request.requestId;
-        // Duplicate issues can occur for the same request; only use the one with a matching networkRequest.
-        if (issueReqId &&
-          networkRecords.find(req => req.requestId === issueReqId)) {
-          artifact.mixedContent.push(issueDetails);
+    /** @type {Array<keyof LH.Artifacts['InspectorIssues']>} */
+    const keys = Object.keys(artifact);
+    for (const key of keys) {
+      /** @type {`${key}Details` | `wasmCrossOriginModuleSharingIssue`} */
```

Comment:
cc @brendankenny 

![image](https://user-images.githubusercontent.com/4071474/145310341-b4146612-685a-4c30-b820-715db1fab2c3.png)


> A 'const' assertions can only be applied to references to enum members, or string, number, boolean, array, or object literals.ts(1355)

too bad

Reply:
Just making sure there was nothing you wanted here before I resolve the comment.